openedx/enterprise-subsidy
# feat: persist the lms user id of JWT-auth'd users

### Description
Uses edX DRF extensions to populate a new `core.User.lms_user_id` field from a JWT.

<img width="1204" alt="image" src="https://user-images.githubusercontent.com/2307986/232893567-fb2dd833-1aee-499f-88e4-0244ced1734d.png">

### Testing instructions
Make a JWT Authenticated request.
Observe in the core.User Django admin that the corresponding user record has a populated `lms_user_id` field, along with an email and name fields.

### Merge checklist
